packages/components/src/dashicon/index.js

```diff
@@ -901,7 +901,7 @@ export default class Dashicon extends Component {
 			return null;
 		}
 
-		const iconClass = IconClass( this.props );
+		const iconClass = IconClass( icon, className, ariaPressed );
```

Comment:
`IconClass` - it looks like a name of the class but this is a regular function. It should be renamed to `getIconClassName`.